Source/Core/Core/IOS/DeviceStub.h

```diff
@@ -19,8 +19,7 @@ namespace Device
 class Stub final : public Device
 {
 public:
-  Stub(u32 device_id, const std::string& device_name);
-
+  using Device::Device;
```

Comment:
You should probably comment that this is intended to inherit constructors from the Device class.